python/amici/sbml_utils.py

```diff
@@ -0,0 +1,722 @@
+"""
+SBML Utilities
+------------
+This module provides helper functions for SBML files.
+"""
+
+from __future__ import annotations
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from .sbml_import import SbmlImporter
+    from typing import Optional, Union
+
+import itertools as itt
+import xml.dom.minidom
+import libsbml
+import sympy as sp
+
+from sympy.printing.mathml import MathMLContentPrinter
+from sympy.core.parameters import evaluate
+
+from sympy.logic.boolalg import BooleanTrue as spTrue
+from sympy.logic.boolalg import BooleanFalse as spFalse
+
+
+################################################################################
+
+
+sbml_time_symbol = sp.Symbol('time', real=True)
+amici_time_symbol = sp.Symbol('t', real=True)
+
+
+annotation_namespace = 'http://github.com/AMICI-dev/AMICI'
+
+
+class SbmlException(Exception):
+    "Exception class for SBML-related errors."
+    pass
+
+
+################################################################################
+
+
+def createSbmlModel(modelId: str, level: int = 2, version: int = 5):
```

Comment:
Please go for snake_case

Reply:
I used camel case to make it stylistically similar to the `libsbml` API, but would not mind changing

Reply:
i see your point, but let's stick to pep8